### PR TITLE
LPS-126366

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by LocalizedNestedFieldsWebContent#TranslateWebContentToNLanguages

**Comment by emanuelfisilva** [comment]
Thanks for your review @Tim-Cao , but we'll choose to keep these validations separated, and we'll think on a better way to refactor this code.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsWebContent.testcase [hunk 1]
```diff
@@ -593,12 +593,93 @@ definition {
...
+			webContentText = "frFR Text",
+			webContentTitle = "WC WebContent frFR");
+
+		PortletEntry.publish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent English");
+
+		PortletEntry.changeLocale(locale = "pt-BR");
+
+		AssertTextEquals(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsWebContent.testcase:646]: Use exiting macro

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsWebContent.testcase [hunk 2]
```diff
@@ -593,12 +593,93 @@ definition {
...
+			locator1 = "TextInput#TITLE",
+			value1 = "WC WebContent ptBR");
+
+		AssertTextEquals.assertValue(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsWebContent.testcase:650]: Use exiting macro

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3770
View total diff https://github.com/liferay-echo/liferay-portal/pull/3770/commits/ad11add295fcf5ec8bb073e259d2fded1e568478

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**